Update cicd Troubleshooting.md (#6272)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md

The service goes through all .csproj-files contained in the uploaded package, and compares that to the versions running in your left-most cloud environment.
We do this to try to prevent you from downgrading the crucial Umbraco packages used by cloud.

The full error could look like this: 

```
Cannot apply update because the following packages would be downgraded: Package: Umbraco.Cms, Version: 13.4.0 ==> Project file: /Update/src/MyAwesomeProject.Code/MyAwesomeProject.Code.csproj contains lower Version: 13.1.0 .
```

The error tells you which package to look for and which version is currently in your left-most cloud environment. The error also contains the problematic incoming .csproj-file and the version referenced by it.

If there are referenced multiple packages with lower versions in the incomming package, the error will list each.

> We recommend that you align versions of the mentioned package(s) in your csproj-files to the higher version mentioned in the error for that package or a later version. 

Reviewer: ```suggestion
We recommend aligning the package versions in your _.csproj_ files with the higher version mentioned in the error for that package or a later version.
```

If you have orphaned csproj-files you should remove them or rename them. 

> Orphaned would be backup csproj files or files not referenced by the any of the main project-files nor referenced in a .sln-file.

Reviewer: ```suggestion
Orphaned would be backup _.csproj_ files or files not referenced by any of the main project files nor referenced in a _.sln_ file.
```
